app/uk/gov/hmrc/helptosavestub/controllers/UserInfoAPIController.scala

```diff
@@ -68,8 +69,11 @@ class UserInfoAPIController extends BaseController {
         val message = "Could not find Authorization in header"
         Logger.error(message)
         Unauthorized(message)
-      } { token ⇒
-        userInfoGen.seeded(token).fold(
+      }{ token ⇒
+        val headerToken = headers.get("token")
+        val userInfo: Option[UserInfo] =
+          hardCodedData.get(headerToken.get).orElse(userInfoGen.seeded(token))
```

Comment:
Be careful of `.get` on `Option`s - they will throw exceptions if the `Option` is `None`